app/webpacker/components/Panel/pages/RunValidatorsPage/ValidationListView.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment, useMemo } from 'react';
+import _ from 'lodash';
+import { Header, List, ListItem } from 'semantic-ui-react';
+import I18n from '../../../../lib/i18n';
+import { competitionUrl } from '../../../../lib/requests/routes.js.erb';
+
+const headingPrefixForType = (type) => {
+  switch (type) {
+    case 'info': return 'Information for';
+    case 'warning': return 'Warnings detected in';
+    case 'error': return 'Errors detected in';
+    default: return 'Unknown detected in';
```

Comment:
Change this text to make it more obvious something wrong happened. Something along the lines `Please contact WST because an unknown response (${type}) was given in`